internal/storage/storage.go

```diff
@@ -144,3 +147,11 @@ func (ms *MemStorage) SaveMetricsList(metricsList *models.MetricsList) {
 		}
 	}
 }
+
+func (ms *MemStorage) Ping(ctx context.Context) error {
+	if ctx == nil {
```

Comment:
CHANGE REQUEST: You don't need to check it

Reply:
\+